specification/components/schemas/extensions/PAFAddressKey.yaml

```diff
@@ -38,7 +38,7 @@ properties:
                 code:
                   type: string
                   description: Address Key system. Always 'PAF'.
-                  default: "PAF"
+                  enum: ["PAF"]
```

Comment:
By defining these as `enum`, we ensure we correctly validate UPRNs against the UPRN _and_ return the correct error when they don't.

Spine's schema's use `const` which `openapi 3.0` do not make available - https://swagger.io/docs/specification/data-models/keywords